.circleci/config.yml

```diff
@@ -32,7 +32,7 @@ commands:
             curl -o /root/.mujoco/mjkey.txt ${MUJOCO_KEY}
             md5sum /root/.mujoco/mjkey.txt
             # Only create venv if it's not been restored from cache
-            [[ -d /venv ]] || USE_MPI=True ./ci/build_venv.sh /venv
+            [[ -d /venv ]] || ./ci/build_venv.sh /venv
```

Comment:
Did you mean to also remove USE_MPI=True? Not sure what this has to do with scipy or the issue you posted

Reply:
it's dead code, we do not ever install `mpi4py` now. But I should mention it in the PR.